xmtp_proto/Cargo.toml

```diff
@@ -24,7 +24,7 @@ grpc = ["tonic"]
 # @@protoc_deletion_point(features)
 # This section is automatically generated by protoc-gen-prost-crate.
 # Changes in this area may be lost on regeneration.
-proto_full = ["xmtp-identity","xmtp-identity-api-v1","xmtp-identity-associations","xmtp-keystore_api-v1","xmtp-message_api-v1","xmtp-message_contents","xmtp-mls-api-v1","xmtp-mls-database","xmtp-mls-message_contents","xmtp-mls_validation-v1"]
+proto_full = ["xmtp-identity","xmtp-identity-api-v1","xmtp-identity-associations","xmtp-keystore_api-v1","xmtp-message_api-v1","xmtp-message_contents","xmtp-mls-api-v1","xmtp-mls-database","xmtp-mls-message_contents","xmtp-mls_validation-v1","xmtp-xmtpv4"]
```

Comment:
unsure on whether or not the v4 stuff should be included. and if so, how to exclude it.

Reply:
We will need these for client changes eventually. It might be churny to include before we start on the client changes, but not a big deal either way